packages/circuits/Makefile

```diff
@@ -2,7 +2,7 @@
 # VARIABLES
 
 # List of powersOfTau trusted setup files here: https://github.com/iden3/snarkjs/blob/master/README.md
-SETUP_FILE=powersOfTau28_hez_final_19.ptau
+SETUP_FILE=powersOfTau28_hez_final_18.ptau
```

Comment:
How much size is the witness file to be downloaded on the frontend now? With the 10x reduction in constraints we achieve, couldn't we go into an even smaller file?

Does the powers of tau file actually affect witness file size? Or just the maximum number of constraint that can be handled?

Reply:
Yes, we can use a smaller file, just made the update.

As far as I know, the ptau file does not affect the witness file size, just the maximum number of constraints. But a smaller ptau means a smaller zkey file, which is still better on the frontend.